packages/frontend/src/components/ui/navbar/index.tsx

```diff
@@ -39,25 +42,25 @@ const navbarStyles = cva(
     }
 );
 
-const navStyles = cva([], {
-    variants: {
-        isOpen: {
-            true: ["absolute flex flex-col top-28 left-0 px-6 py-16  w-full"],
-            false: ["hidden md:flex"],
-        },
-    },
-});
+// const navStyles = cva([], {
+//     variants: {
+//         isOpen: {
+//             true: ["absolute flex flex-col top-28 left-0 px-6 py-16  w-full"],
+//             false: ["hidden md:flex"],
+//         },
+//     },
+// });
 
-const navLinksStyles = cva(["flex"], {
-    variants: {
-        isOpen: {
-            true: ["flex-col items-start space-y-8 relative"],
-            false: [
-                "items-center space-x-8 left-1/2 absolute transform -translate-x-1/2 -translate-y-1/2",
-            ],
-        },
-    },
-});
+// const navLinksStyles = cva(["flex"], {
+//     variants: {
+//         isOpen: {
+//             true: ["flex-col items-start space-y-8 relative"],
+//             false: [
+//                 "items-center space-x-8 left-1/2 absolute transform -translate-x-1/2 -translate-y-1/2",
+//             ],
+//         },
+//     },
+// });
```

Comment:
removing this code, breaks the open menu on mobile.

<img width="556" alt="image" src="https://user-images.githubusercontent.com/5664434/220164132-c7ab1476-5533-4bb6-a17b-6c95fd6d2391.png">